### ISSUE TITLE
[CoE Starter Kit - BUG] Child Admin | Sync Logs flow 'use this connection' not visible in Microsoft Dataverse connection

### ISSUE THREAD

**Issue by dchrist27** [microsoft/coe-starter-kit#3820]
### Describe the issue

This issue is the same as previously reported #3376.    I've tried the following and I still do not have the ability to set 'use this connection'

-delete solution layer and attempted to reconfigure Run Only Users  (failed)
-delete the Audit Logs solution then reimport  (failed)
-delete the existing Dataverse connection and also deleted/imported the Audit Logs solution (failed)

![image](https://user-images.githubusercontent.com/52799471/193290272-0deb5c33-adb3-4ae9-adc8-f4858ea71d3a.png)

Any ideas on how to fix this issue?

### What solution are you experiencing the issue with?

Audit Log

### What solution version are you using?

2.10

### What app or flow are you having the issue with?

Child Admin | Sync Logs

**Comment by manuelap-msft** [comment]
Can you try following these steps:
1. Navigate to make.powerapps.com
2. Navigate to Dataverse > Connections
3. + New connection
4. Add a connection to Microsoft Dataverse connector
5. Go back to the run-only users list of this cloud flow and see if you can update the connector there

**Comment by albertocaa1317** [comment]
Hi @manuelap-msft,
I am configuring a new CoE environment from scratch with Sept'22 versión and I have the same issue.

I already had follow the steps that you propose but neither I can't select a Datavese connection.

**Comment by albertocaa1317** [comment]
Hi @Jenefer-Monroe 
Exactly. Despite of I had created a new Dataverse connection It doesn't appears in the selector.
Do you suggest to open a support case?

**Comment by dchrist27** [comment]
Since I was starting a brand new environment...I decided to create another new environment and step through all the core configs, then import the audit logs solution.  Unfortunately I am seeing the same issue.

This is what I see when I first go into Run Only Users of the Child flow.

![image](https://user-images.githubusercontent.com/52799471/193623917-f87c1590-c782-4e78-aeca-50367ab39453.png)

Just strange because I see all 3 connections in this flow.

![image](https://user-images.githubusercontent.com/52799471/193624117-6f446558-7d84-4388-9021-24c1d559ec78.png)

**Comment by Jenefer-Monroe** [comment]
Unfortunately yes. Please let me know if you get suggestions on something we could have done better here!

**Comment by albertocaa1317** [comment]
Support case opened. I hope give you news asap.

**Comment by dchrist27** [comment]
Hi @Jenefer-Monroe or @manuelap-msft can you comment on my last post/question?

I can create a (new) Dataverse connector but it does not show up in run-only users dropdown.

I was debating on changing the connection reference for the 4 Dataverse actions within the Switch step of the Child flow.   Or would you suggest another approach?   And to confirm wouldn’t this add a solution layer that would have to be removed when I update the solution in the future?

**Comment by Jenefer-Monroe** [comment]
I'm afraid you have hit some kind of product issue here. You should see the connection offered in the run only users dropdown and the fact that you dont indicates a product bug of some kind.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team

**Comment by petepuu** [comment]
@Jenefer-Monroe @albertocaa1317 @dchrist27 I noticed the same issue with October version as well and heard from 2 of my colleagues that they saw the same issue in the clients. We did some troubleshooting and the fix in my case at least was very easy. Just start the flow with that configuration Dataverse having empty connection and Office 365 Management API and Outlook run-only user. After you have started the flow, those connections are fixed automatically to have your CoE service account selected
